codegenerator/cli/templates/static/codegen/src/Utils.res

```diff
@@ -219,21 +219,96 @@ module Schema = {
   let coerceToJsonPgType = schema => {
     schema->S.preprocess(s => {
       switch s.schema->S.classify {
-        | Literal(Null(_)) => {serializer: _ => %raw(`"null"`)}
-        | Null(_)
-        | Bool => {serializer: unknown => {
-          if unknown === %raw(`null`) {
-            %raw(`"null"`)
-          } else if unknown === %raw(`false`) {
-            %raw(`"false"`)
-          } else if unknown === %raw(`true`) {
-            %raw(`"true"`)
-          } else {
-            unknown
-          }
-        }}
-        | _ => {}
+      | Literal(Null(_)) => {serializer: _ => %raw(`"null"`)}
+      | Null(_)
+      | Bool => {
+          serializer: unknown => {
+            if unknown === %raw(`null`) {
+              %raw(`"null"`)
+            } else if unknown === %raw(`false`) {
+              %raw(`"false"`)
+            } else if unknown === %raw(`true`) {
+              %raw(`"true"`)
+            } else {
+              unknown
+            }
+          },
+        }
+      | _ => {}
       }
     })
   }
 }
+
+module Set = {
+  type t<'value>
+
+  /*
+   * Constructor
+   */
+  @ocaml.doc("Creates a new `Set` object.") @new
+  external make: unit => t<'value> = "Set"
+
+  @ocaml.doc("Creates a new `Set` object.") @new
+  external fromEntries: array<'value> => t<'value> = "Set"
```

Comment:
Out of interest why rebind this whole module? Just to be explicit about what the bindings are?

Reply:
To prevent the case when you expect it to use the bindings, but then it unexpectedly used `Belt` version instead